### PR TITLE
Add Python LSP Server

### PR DESCRIPTION
**Pull request by maresb**
I didn't have time to test this or to fill out this pull request properly. :cry: Sorry!!!

## Chores

- [ ] linted 
- [ ] tested 
- [ ] documented 
- [x] changelog entry

### CODE REVIEW

**Comment by maresb** [comment]
Addresses https://github.com/python-lsp/python-lsp-server/issues/17

File: python_packages/jupyter_lsp/jupyter_lsp/specs/python_lsp_server.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+            ),
+            dict(
+                display_name="pyls-memestra",
+                install=dict(
+                    pip="pip install pyls-memestra",
+                    conda="conda install pyls-memestra",
+                ),
+            ),
+        ],
+        config_schema=load_config_schema(key),
```
Comment (krassowski) [inline python_packages/jupyter_lsp/jupyter_lsp/specs/python_lsp_server.py:47]: The CI complains about missing `python-lsp-server.schema.json` here, I guess that we need to copy-paste it too (these will start diverging over time I believe).

**Comment by krassowski** [comment]
Thank you for the help @maresb.

**Comment by maresb** [comment]
I was out today, so thanks so much @krassowski for seeing this through!!!

**Pull request merged**